### DISCUSSION TITLE
How can i bold text while sending tweet

### DISCUSSION THREAD

**Discussion by vishnu1991** [plhery/node-twitter-api-v2#476] [Q&A]
Searched docs but was not able to find if there is a way to send **bold** text in twitter even tried markdown formatting and html formatting but both doesn't work.
Any help will be appreciated.
Thanks

**Comment by satyamy846** [comment]
I don't think twitter text supports this bold text through the api
see the doc - https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet
